app/settings.py

```diff
@@ -37,6 +34,16 @@ def get_key(key_name):
 EQ_SESSION_TIMEOUT = int(os.getenv('EQ_SESSION_TIMEOUT', '28800'))
 EQ_SECRET_KEY = os.getenv('EQ_SECRET_KEY', os.urandom(24))
 
+# keys for the RRM token exchange
+EQ_RRM_PUBLIC_KEY = get_key(os.getenv('EQ_RRM_PUBLIC_KEY', "./jwt-test-keys/rrm-public.pem"))
+EQ_SR_PRIVATE_KEY = get_key(os.getenv('EQ_SR_PRIVATE_KEY', "./jwt-test-keys/sr-private.pem"))
```

Comment:
Might be worth naming these now to reflect their usage e.g. This is the user authentication encryption private key, the one above is the user authentication signing public key and the one below is the submission signing private key etc?